util/ipblocker/ipblocker.go

```diff
@@ -0,0 +1,56 @@
+package ipblocker
+
+import (
+	"net"
+	"sync"
+)
+
+type IPBlocker struct {
+	cidrs []*net.IPNet
+}
+
+func New(blackListCidr []string) (*IPBlocker, error) {
```

Comment:
To be in harmony with the rest of code, lets rename it to "NewIPBlocker"

Reply:
It's standard name base on go effective.

```go
ipblocker.New()
```

Package name make explicit it, not required another things.

Because package encapsulated.

Reply:
@Ja7ad Please update other packages.